server/controllers/users.js

```diff
@@ -150,6 +150,41 @@ class UserController {
       next(error);
     }
   }
+
+  /**
+  * Get users and their corresponding profiles
+  * @async
+  * @param {object} req - Request object
+  * @param {object} res - Response object
+  * @param {object} next The next middleware
+  * @return {json} Returns json object
+  * @static
+  */
+  static async getUsers(req, res, next) {
+    try {
+      const users = await User.findAll({
+        attributes: { exclude: ['createdAt', 'updatedAt', 'password', 'email'] },
```

Comment:
please change to include only attributes needed.

I see some unneeded attributes returned.

same thing for the profile model. (there's no need including `userId` in the profile aspect for instance